airctrl/airctrl.py

```diff
@@ -116,7 +116,11 @@ def _get(self, url):
             self._get_key()
             return self._get_once(url)
 
-    def _dump_status(self, status, debug=False):
+    def _dump_status(self, status, use_json=False, debug=False):
+        if use_json and not debug:
+            pprint.pprint(status)
```

Comment:
I think you can reuse the `debug` option, just make it print to `stderr` like this:
```
pprint.pprint(status, stream=sys.stderr)
```
this way you will get only the json output on `stderr` and you can feed to it to other scripts. Make sense?

Reply:
Hi @rgerganov, sorry for the delay. I don't think the JSON output should be treated as an error stream as it has nothing to do with it. I understand adding another parameter is not the best option though… Maybe I can create another Pull Request without the JSON stuff and I can port your code to Javascript so that I don't need to run it as a child process, but I would need a bit of your help for it if that's possible.

Thanks anyway!

Reply:
well, `stderr` is intended for both error messages *and* diagnostic information which is kind of what the jsout output is. Anyway, let's do the `ddp` and `uil` first.